blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,173 @@
+
+    cr> GRANT READ ON TABLE school.courses TO wolfgang;
+    GRANT OK, 1 row affected (... sec)
+
+To grant ``WRITE`` privilege to user riley on the tables books, and school.courses we can
+use the following syntax::
+
+    cr> GRANT WRITE ON TABLE books, school.courses TO riley;
+    GRANT OK, 1 row affected (... sec)
+
+Granting ``WRITE`` privilege on a table to a user, indicates that this user is allowed to run
+``INSERT``, ``ALTER TABLE`` ``DELETE`` and ``DROP TABLE`` statements on this table.
+
+Users having ``WRITE`` privilege on a table, will still need ``READ`` privilege to be able to run
+``SELECT`` statements on that table.
+
+To grant ``ALL`` privileges to user riley on table books, we can use the following syntax::
+
+    cr> GRANT ALL ON TABLE books TO riley;
+    GRANT OK, 2 rows affected (... sec)
+
+``ALL`` implies all the currently available privileges, namely ``READ`` and ``WRITE``.
+
+The superuser can also grant privileges on all tables in a schema, like this::
+
+    cr> GRANT READ ON ALL TABLES IN SCHEMA school TO riley;
+    GRANT OK, 2 row affected (... sec)
+
+Privileges can be granted to multiple users in the same statement, like so::
+
+    cr> GRANT READ ON ALL TABLES IN SCHEMA school TO kala, will;
+    GRANT OK, 6 row affected (... sec)
+
+
+``ALL TABLES IN SCHEMA`` implies only the tables that currently exist in a schema.
+
+Currently, there is no option to grant privileges on all tables of the cluster.
+
+.. warning:
+
+    It is not possible to ``GRANT`` or ``REVOKE`` privileges on ``information_schema``
+    tables.
+
+``REVOKE``
+----------
+
+To revoke a privilege that was previously granted to a user use the
+:doc:`/sql/reference/revoke` SQL statement, for example::
+
+    cr> REVOKE WRITE ON TABLE books FROM riley;
+    REVOKE OK, 1 row affected (... sec)
+
+If a user with the username specified in the SQL statement does not exist the
+statement returns an error::
+
+    cr> REVOKE WRITE ON TABLE books FROM layla;
+    UserDoesNotExistException ...
+
+It is not possible to ``REVOKE`` privileges from the built-in superuser ``crate``.
+
+The `READ`` privileges that were previously granted to the user riley on all
+tables of the school schema, can be revoked like this::
+
+    cr> REVOKE READ ON ALL TABLES IN SCHEMA school FROM riley;
+    REVOKE OK, 3 row affected (... sec)
+
+``REVOKE ALL`` or ``REVOKE ALL PRIVILEGES`` will revoke all privileges on a table
+that were previously granted to a user, it can be used like this::
+
+    cr> REVOKE ALL ON ALL TABLES IN SCHEMA school FROM kala;
+    REVOKE OK, 3 row affected (... sec)
+
+    cr> REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA school FROM will;
+    REVOKE OK, 3 row affected (... sec)
+
+
+List Privileges
+---------------
+
+CrateDB exposes table privileges via the read-only
+``information_schema.table_privileges`` system table.
+
+To list all existing privileges query that table::
```

Comment:
I would prefer something like:
`By querying the table you can get info about the existing privileges. E.g.::`

@nslater what's your opinion?